crates/cli/src/lib.rs

```diff
@@ -118,11 +119,14 @@ async fn update(context: Context<impl Environment>) -> anyhow::Result<()> {
         .context_path
         .join(configuration::CONFIGURATION_FILENAME);
     let input: configuration::RawConfiguration = {
-        let reader = fs::File::open(&configuration_file_path)?;
-        serde_json::from_reader(reader)?
```

Comment:
Seems like `from_reader` is a bit slow and rubbish anyway, so this seems fine? https://github.com/serde-rs/json/issues/160